script/tool/lib/src/fetch_gradle_deps_command.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file/file.dart';
+import 'package:platform/platform.dart';
+
+import 'common/core.dart';
+import 'common/gradle.dart';
+import 'common/package_looping_command.dart';
+import 'common/plugin_utils.dart';
+import 'common/process_runner.dart';
+import 'common/repository_package.dart';
+
+/// Run 'gradlew dependencies'.
+///
+/// See https://docs.gradle.org/6.4/userguide/core_dependency_management.html#sec:dependency-mgmt-in-gradle.
+class FetchGradleDeps extends PackageLoopingCommand {
```

Comment:
Is there any way we can do a shared cache, or does Gradle caching only work at the project level?

Reply:
Gradle uses a shared cache as a default.